src/Components/sidebar.module.css

```diff
@@ -0,0 +1,51 @@
+li{
+    list-style-type: none;
+}
+
+.navbar {
+    background: linear-gradient(to right, #ffffff, #f0f0f0);
+    max-width: 25%;
+    padding: 0.2rem;
+    height: 100vh;
+}
+
+.text {
+    font-size: 1.5rem;
+    padding: 0.5rem 0rem 0rem 0.9rem;
+    font-family: 'Poppins', sans-serif;
+}
+
+.navbar a {
+    color: #01001e;
+    text-decoration: none;
+    transition: color 0.3s ease;
```

Comment:
Please take a look, doesnt do anything.

Reply:
When I remove it it does change the color, ill be happy to check it again together.